rotkehlchen/data_import/importers/bitcoin_tax.py

```diff
@@ -0,0 +1,278 @@
+        )
+        action = csv_row['Action']
+        memo = f"Imported description from bitcoin tax: {csv_row['Memo']}" if csv_row['Memo'] else ''  # noqa: E501
+
+        base_asset_balance = AssetBalance(base_asset, Balance(base_asset_amount, ZERO))
+        fee_asset_balance = None
+        if fee_amount != ZERO and fee_asset is not None:
+            fee_asset_balance = AssetBalance(fee_asset, Balance(fee_amount, ZERO))
+
+        if csv_type == 'trades':
+            quote_asset = asset_resolver(csv_row['Currency'])
+            quote_asset_amount = deserialize_asset_amount(csv_row['Cost/Proceeds'])
+            quote_asset_balance = AssetBalance(quote_asset, Balance(quote_asset_amount, ZERO))
+            self._consume_trade_event(
+                cursor=cursor,
+                csv_row=csv_row,
+                event_identifier=event_identifier,
+                timestamp=timestamp,
+                location=location,
+                action=action,
+                base_asset_balance=base_asset_balance,
+                quote_asset_balance=quote_asset_balance,
+                fee_asset_balance=fee_asset_balance,
+                memo=memo,
+            )
+            return
+        self._consume_income_spending_event(
+            cursor=cursor,
+            csv_row=csv_row,
+            event_identifier=event_identifier,
+            timestamp=timestamp,
+            location=location,
+            action=action,
+            asset_balance=base_asset_balance,
+            fee_asset_balance=fee_asset_balance,
+            memo=memo,
+        )
+
+    def _import_csv(self, cursor: DBCursor, filepath: Path, **kwargs: Any) -> None:
+        """
+        May raise:
+        - InputError if one of the rows is malformed
+        """
+        with open(filepath, encoding='utf-8-sig') as csvfile:
+            data = csv.DictReader(csvfile)
+            csv_type = determine_csv_type(data)
+            for _, row in enumerate(data):
+                try:
+                    self._consume_event(cursor, row, csv_type, **kwargs)
+                except UnknownAsset as e:
+                    self.db.msg_aggregator.add_warning(
+                        f'During BitcoinTax CSV import found action with unknown '
+                        f'asset {e.identifier}. Ignoring entry',
+                    )
+                    continue
+                except DeserializationError as e:
+                    self.db.msg_aggregator.add_warning(
+                        f'Deserialization error during BitcoinTax CSV import. '
+                        f'{e!s}. Ignoring entry',
+                    )
+                    continue
+                except UnsupportedCSVEntry as e:
+                    self.db.msg_aggregator.add_warning(str(e))
+                    continue
+                except KeyError as e:
+                    raise InputError(f'Could not find key {e!s} in csv row {row!s}') from e
```

Comment:
There is nothing afterwards so all these `continue` are not needed, are they?

```suggestion
                except UnknownAsset as e:
                    self.db.msg_aggregator.add_warning(
                        f'During BitcoinTax CSV import found action with unknown '
                        f'asset {e.identifier}. Ignoring entry',
                    )
                except DeserializationError as e:
                    self.db.msg_aggregator.add_warning(
                        f'Deserialization error during BitcoinTax CSV import. '
                        f'{e!s}. Ignoring entry',
                    )
                except UnsupportedCSVEntry as e:
                    self.db.msg_aggregator.add_warning(str(e))
                except KeyError as e:
                    raise InputError(f'Could not find key {e!s} in csv row {row!s}') from e
```